language/move-lang/src/shared/mod.rs

```diff
@@ -243,10 +243,10 @@ pub const fn sp<T>(loc: Loc, value: T) -> Spanned<T> {
 /// Macro used to create a tuple-like pattern match for Spanned
 macro_rules! sp {
     (_, $value:pat) => {
-        Spanned { value: $value, .. }
+        Spanned { value: $value, loc: _ }
     };
     ($loc:pat, _) => {
-        Spanned { loc: $loc, .. }
+        Spanned { loc: $loc, value: _ }
```

Comment:
Mega nit. unrelated change to the PR. Its fine here, but try to avoid these if possible :)

Reply:
Ah sorry this was merely a way to workaround a bug in rust-analyzer for myself. Won't include this in the final PR.